src/intl/fr/glossary.json

```diff
@@ -128,7 +128,7 @@
   "execution-layer-term": "Couche d'exécution",
   "execution-layer-definition": "La couche d'exécution d'Ethereum est le réseau des <a href=\"/glossary/#execution-client\">clients d'exécution</a>.",
   "eoa-term": "Compte externe (EOA)",
-  "eoa-definition": "Les comptes détenus en externe (EOA) sont des <a href=\"/glossary/#account\">comptes</a> contrôlés par des <a href=\"/glossary/#private-key\">clés privées</a>, généralement générées à l'aide d'une <a href=\"/glossary/#hd-wallet-seed\">phrase de récupération</a>. Contrairement aux contrats intelligents, les comptes détenus en externe sont des comptes auxquels aucun code n'est associé. En général, ces comptes sont gérés par un <a href=\"/glossary/#wallet\">portefeuille</a>.",
+  "eoa-definition": "Les comptes détenus en externe (EOA) sont des <a href=\"/glossary/#account\">comptes</a> contrôlés par des <a href=\"/glossary/#private-key\">clés privées</a>, généralement générées à l'aide d'une <a href=\"/glossary/#recovery-phrase\">phrase de récupération</a>. Contrairement aux contrats intelligents, les comptes détenus en externe sont des comptes auxquels aucun code n'est associé. En général, ces comptes sont gérés par un <a href=\"/glossary/#wallet\">portefeuille</a>.",
```

Comment:
Was `recovery-phrase` added recently? I'm not finding it in the glossary

Reply:
I see `recovery-phrase-term` and `recovery-phrase-definition` in `/en/glossary-tooltip.json` and `/en/glossary.json`

Was merged last week https://github.com/ethereum/ethereum-org-website/pull/12236, but isnt in production yet.